### PR TITLE
testing/apache2-mod-security: new aport

### PR DESCRIPTION
**Pull request by esler**
https://www.modsecurity.org/
ModSecurity is an open source, cross platform web application firewall (WAF) engine for Apache

### CODE REVIEW

File: testing/apache2-mod-security/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+_realname=mod_security2
+pkgver=2.9.3
+pkgrel=0
+pkgdesc="Security Module for Apache2"
+url="https://www.modsecurity.org/"
+arch="all"
+license="Apache-2.0"
+depends="apache2 pcre libxml2"
+makedepends="autoconf automake libtool linux-headers apache2-dev pcre-dev libxml2-dev"
+install=""
```
Comment (andypost) [inline testing/apache2-mod-security/APKBUILD]: please remove as unneeded

File: testing/apache2-mod-security/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+check() {
+        cd "$builddir"
+        make -j1 test
+}
+
+package() {
+	cd "$builddir"
+	install -Dm755 "apache2/.libs/$_realname.so" "$pkgdir/usr/lib/apache2/$_realname.so"
+        install -Dm644 "$srcdir/$pkgname".conf \
```
Comment (andypost) [inline testing/apache2-mod-security/APKBUILD]: Nit: something wrong with indent

File: testing/apache2-mod-security/APKBUILD (hunk 1 above)
Comment (esler) [reply testing/apache2-mod-security/APKBUILD]: it's gone

File: testing/apache2-mod-security/APKBUILD (hunk 2 above)
Comment (esler) [reply testing/apache2-mod-security/APKBUILD]: tabs and spaces were mixed

**Comment by andypost** [comment]
Now looks nice, the only thing is to squash commits to single one `git rebase -i HEAD~3`

**Comment by esler** [comment]
> Now looks nice, the only thing is to squash commits to single one `git rebase -i HEAD~3`

Done 👍

File: testing/apache2-mod-security/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+pkgname=apache2-mod-security
+_pkgreal=modsecurity
+_realname=mod_security2
+pkgver=2.9.3
+pkgrel=0
+pkgdesc="Security Module for Apache2"
+url="https://www.modsecurity.org/"
+arch="all"
+license="Apache-2.0"
+depends="apache2 pcre libxml2"
```
Comment (andypost) [inline testing/apache2-mod-security/APKBUILD]: I bet `pcre libxml2` will be added by build system... see in travis
```
...
>>> apache2-mod-security*: Tracing dependencies...
	apache2
	pcre
	libxml2
	so:libapr-1.so.0
	so:libaprutil-1.so.0
	so:libc.musl-x86_64.so.1
	so:libpcre.so.1
	so:libxml2.so.2
```

Also not clear why travis reports `>>> WARNING: apache2-mod-security*: Redundant /usr/lib in rpath found`

File: testing/apache2-mod-security/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+makedepends="autoconf automake libtool linux-headers apache2-dev pcre-dev libxml2-dev"
+subpackages="$pkgname-doc"
+source="https://github.com/SpiderLabs/ModSecurity/releases/download/v$pkgver/$_pkgreal-$pkgver.tar.gz
+	$pkgname.conf"
```
Comment (kunkku) [inline testing/apache2-mod-security/APKBUILD]: Should the configuration file be named security.conf to follow the pattern in the main package?

**Pull request closed without merging**